.golangci.yml

```diff
@@ -117,14 +117,14 @@ linters:
     - misspell
     - nakedret
     - staticcheck
-    - structcheck
     - stylecheck
     - typecheck
     - unconvert
-    - unparam
     - unused
     - varcheck
     - whitespace
+  disable:
+    - structcheck
```

Comment:
Disable? Because?

Reply:
Because structcheck is disabled in go1.18.  When using the latest version of golangci-lint, the following warning appears
```
make lint
...
WARN [linters context] structcheck is disabled because of go1.18. You can track the evolution of the go1.18 support by following the https://github.com/golangci/golangci-lint/issues/2649.
```